recipe/patch_yaml/glum.yaml

```
@@ -0,0 +1,11 @@
+#  ensure that breaking changes from tabmat 4.0.0
+#  do not interfere with older versions of glum
+if:
+  name: glum
+  timestamp_lt: 1713800640000
+  version_lt: 2.6.0
+  has_depends:
+    - tabmat*
+  not_has_constrains: tabmat < 4.0.0
+then:
+  - add_constrains: tabmat < 4.0.0
```

This should be `depends` and not `constraints`

Small nit:
```suggestion
  - add_constrains: tabmat <4
```

---

Thanks for the review. It should work with `depends` now.